app/utils/date.ts

```diff
@@ -53,3 +53,14 @@ export const getTimeLeft = ({ after, until }: { after: number; until: number })
   const sLeft = (until - dateNow) / 1000
   return secondsToDDMMSS(sLeft)
 }
+
+// e.g. 1747691078 -> "2025-05-19 15:44"
+export function formatUnixTimestampYMDHM(timestampInSeconds: number) {
```

Comment:
is this not like .toISOString ?

`new Date(1747691078 * 1000).toISOString().slice(0, 16).replace('T', ' ')`